packages/db/schema.ts

```diff
@@ -235,9 +234,9 @@ export const invites = pgTable(
 );
 
 export const invitesRelations = relations(invites, ({ one }) => ({
-	invitee: one(users, {
+	invitee: one(userCommonData, {
```

Comment:
I think we can change this one to `userHackerData` as well as these invites should be to join teams which should only be allowed by hackers.